exotica_core/src/kinematic_tree.cpp

```diff
@@ -1008,7 +1083,47 @@ void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
     }
     for (int i = 0; i < 6; ++i)
     {
+        // Set velocity and acceleration to +inf
         controlled_joints_[i].lock()->joint_limits = {lower[i], upper[i]};
+        controlled_joints_[i].lock()->velocity_limit = {inf};
+        controlled_joints_[i].lock()->acceleration_limit = {inf};
+    }
+
+    UpdateJointLimits();
+}
+
+void KinematicTree::SetPlanarBaseLimitsPosXYEulerZ(
+    const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration)
+{
+    bool velocity_acceleration_to_inf = false;
```

Comment:
Same comments apply to these methods as above: To remove the this logic and only override what is being passed in